docs/content/configuration/csi_ug_config_create_secret.md

```diff
@@ -30,7 +32,7 @@ Use one of the following procedures to create and apply the secret:
     `kubectl apply -f demo-secret.yaml`
     
 
-     The `secret/<NAME> created` message is emitted.
+     The `secret/<demo-secret> created` message is emitted.
```

Comment:
why have we defined here `demo-secret` and not `secret-name` as in other places (`pvc-name` and `statefulset-name`)?

Reply:
@oriyarde  which is correct here?

Reply:
done

Reply:
I would also align the `<>` values in the [`kubectl create secret generic`](https://github.com/IBM/ibm-block-csi-driver/blob/e11538cc8eade54e341daf563052d3bf00418b36/docs/content/configuration/csi_ug_config_create_secret.md#creating-an-array-secret-via-command-line) command, to something like:
```
kubectl create secret generic demo-secret --from-literal=username=demo-username --from-literal=password=demo-password --from-literal=management_address=demo-management-address -n demo-namespace
```
note there is currently a missing space right before `--from-literal=management_address`.

Reply:
DONE